nncf/torch/quantization/algo.py

```diff
@@ -227,22 +226,6 @@ def _get_default_qconfig(self, constraints: QuantizationConstraints = None):
             qconfig = constraints.apply_constraints_to(qconfig)
         return qconfig
 
-    def _should_consider_scope_for_group(self, node_name: NNCFNodeName, group: QuantizerGroup) -> bool:
```

Comment:
This was a well-defined method before, do not inline it.

Reply:
Looks like duplication of `should_consider_scope`